src/main/java/net/sf/jabref/logic/bibtexkeypattern/BibtexKeyPatternUtil.java

```diff
@@ -519,6 +521,13 @@ public static String applyModifiers(String label, List<String> parts, int offset
                     }
                     resultingLabel = abbreviateSB.toString();
 
+                } else if ("camel".equals(modifier)) {
+                    // CamelCasingOfEveryWord
+                    Title title = new Title(resultingLabel);
+                    title.getWords().stream().forEach(Word::toUpperFirst);
```

Comment:
If getWord returns a list you can also use the new fancy replaceAll overload instead of the forEach.  But in this case it doesn't really matter ;)

https://docs.oracle.com/javase/8/docs/api/java/util/List.html#replaceAll-java.util.function.UnaryOperator-